Add Comunidade Meshtastic Portugal to local-groups.mdx

l-margiela opened the pull request:
<!-- Add or remove sections as needed -->
## What did you change
<!-- Describe what your changes will do if merged -->

Adds the first unofficial group for Portugal. 🎉 

## Why did you change it
<!-- Be sure to link any relevant changes such as other PRs, issues, or Discord convos -->

~~Noticed there isn't any so I decided to start one. I'm living in Portugal and I'd like to extend the mesh a bit more, especially on the north of the country.~~ There's one and it's flourishing! Removed the new group; let's add the existing community instead.

## Screenshots

See the rich diff.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/meshtastic/meshtastic?pullRequest=1854) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/meshtastic/meshtastic?pullRequest=1854) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/meshtastic/meshtastic?pullRequest=1854) it.</sub>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/meshtastic/meshtastic?pullRequest=1854) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/meshtastic/meshtastic?pullRequest=1854) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/meshtastic/meshtastic?pullRequest=1854) it.</sub>

pdxlocations commented:
<img width="235" alt="Screenshot 2025-05-11 at 7 54 34 AM" src="https://github.com/user-attachments/assets/33ca35a2-cb8b-4b07-81f0-39d7de3ebe85" />

Please change "Meshtastic Portugal" to something like "Portugal Meshtastic Users Group" so you don't imply official sponsorship.  Thanks!

l-margiela commented:
@pdxlocations Thanks a lot for the review! Change of plans, though -- turns out my community kick starting was a bit premature. There's already a Portuguese community that exists for quite some time and is very active. Changed this PR:

1. Removed the group I just created
2. Added Comunidade Meshtastic Portugal

Let me know if it's fine to merge it in this form! Thanks a lot

l-margiela commented:
@pdxlocations Changed the URL to `https://t.me/comunidademeshtasticpt`. Please let me know if it's good now
